docs/metrics/promotheus-grafana.md

```diff
@@ -144,63 +144,67 @@ To monitor a Citrix ADC CPX ingress device, the Citrix ADC metrics exporter is a
 
 ```YAML
 ---
-apiVersion: extensions/v1beta1
```

Comment:
same as above, refer Can we make changes in all the exporter file as per the example here : https://github.com/citrix/citrix-k8s-ingress-controller/tree/master/metrics-visualizer#configuring-citrix-adc-metrics-exporter
Looks, like these canary and other examples were not updated. Let me know if case of any doubt.